src/test/scala/uk/gov/hmrc/perftests/dds/requests/DigitalDisclosureRequests.scala

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.perftests.dds.requests
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import io.gatling.http.request.builder.HttpRequestBuilder
+import uk.gov.hmrc.perftests.dds.Configuration
+
+object DigitalDisclosureRequests extends Configuration {
+
+  val digitalDisclosureRoute = s"$digitalDisclosureUrl/digital-disclosure"
```

Comment:
This is an example of what I mean in [this comment](https://github.com/hmrc/digital-disclosure-service-performance-tests/pull/3/files#r1048670023). You should define a route rather than include `/notification` in every single request. For example, if the service ever decided to change `/notification` to `/notifications` you only have to update one line rather than every single request.